x-pack/plugins/osquery/public/common/hooks/use_discover_link.tsx

```diff
@@ -7,7 +7,7 @@
 
 import { useCallback, useEffect, useState } from 'react';
 
-import { FilterStateStore } from '@kbn/data-plugin/common';
+import { FilterStateStore } from '@kbn/es-query';
```

Comment:
I think we should refactor `FilterStateStore` from enum to a type so that osquery (and probably other places) avoid importing '@kbn/es-query' into their bundle just for this

Reply:
https://github.com/elastic/kibana/issues/134648